bot/src/bounded-contexts/analytics/domain/services/processors/CorpsDataProcessor.ts

```
@@ -0,0 +1,347 @@
+                      'Unknown Corporation';
+      
+      if (!corpStats.has(corpName)) {
+        corpStats.set(corpName, {
+          kills: 0,
+          losses: 0,
+          killValue: 0,
+          lossValue: 0
+        });
+      }
+      
+      const stats = corpStats.get(corpName)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)